checkov/sca_package_2/runner.py

```diff
@@ -45,8 +46,16 @@ def prepare_and_scan(
             logging.info("The --bc-api-key flag needs to be set to run SCA package scanning")
             return None
 
+        if bc_integration.bc_source and bc_integration.bc_source.name in [BCSourceType.VSCODE, BCSourceType.JETBRAINS] \
+                and not bc_integration.is_prisma_integration():
+            logging.info("The --bc-api-key flag needs to be set to Prisma token for SCA scan for vscode or jetbrains extention")
```

Comment:
```suggestion
            logging.info("The --bc-api-key flag needs to be set to a Bridgecrew or Prisma token for SCA scan for vscode or jetbrains extention")
```

Reply:
@tsmithv11 can you help with some proper wording here 😄

Reply:
We need to block Bridgecrew and allow only Prisma